application/rest/casp/getUncompressedPublicKeys.go

```diff
@@ -9,50 +9,65 @@ import (
 	"crypto/tls"
 	"encoding/json"
 	"fmt"
+	"io"
+	"net/http"
+
 	"github.com/persistenceOne/persistenceBridge/application/configuration"
 	"github.com/persistenceOne/persistenceBridge/application/rest/responses/casp"
 	"github.com/persistenceOne/persistenceBridge/utilities/logging"
-	"io"
-	"io/ioutil"
-	"net/http"
 )
 
 func GetUncompressedTMPublicKeys() (casp.UncompressedPublicKeysResponse, error) {
 	return getUncompressedPublicKeys(configuration.GetAppConfig().Tendermint.CoinType)
 }
 
+const ethCoinType = 60
```

Comment:
why declare this?

Reply:
magic number was extracted and name given